src/common/url/check-url.js

```diff
@@ -0,0 +1,12 @@
+export function isValidAssetUrl (string) {
+  /* Note: This API is not available in IE11 and throws in Safari v14- when base is undefined.
+    Hence, this will always return false for those environments.
+  */
+  try {
+    let url = new URL(string)
```

Comment:
try swapping this out to `parseUrl`

Reply:
Doesn't work. This function tries to ensure customer provides a domain:port aka origin; however, `parseUrl` interprets relative paths with the current location as base.